src/services/ParticipantProjectService.ts

```diff
@@ -60,6 +66,78 @@ const get = async (participantProjectId: number): Promise<Response2<ParticipantP
   };
 };
 
+const downloadList = async (search?: SearchNodePayload, sortOrder?: SearchSortOrder): Promise<string | null> => {
+  return (
+    'Project Name,Participant,Cohort,Phase,Country,Region,Restorable Land,Land Use Model Type\r' +
+    'Andromeda,Cartwheel,Cohort 1,Phase 0 - Due Diligence,Ghana,Sub-Saharan Africa,500,Native Forest'
+  );
+};
+
+const list = async (
+  // TODO: remove locale if we are using BE search API
```

Comment:
For this to use BE search API we need to add CohortsSearchTable and ParticipantsSearchTable on the BE.